lib/sidekiq_alive/worker.rb

```diff
@@ -3,17 +3,23 @@
 module SidekiqAlive
   class Worker
     include Sidekiq::Worker
-    sidekiq_options retry: false
```

Comment:
Huh, adding this to the SidekiqAlive::Config class should probably be a PR to arturictus.

Reply:
I think the original is the more typical behavior. If the code works, we could prepare a PR that would make it configurable.

Reply:
yeah, I wasn't particularly clear.  I did mean by 'adding this to the SidekiqAlive::Config class' to make it configurable.  It kind of looks like that was the original intent given the name, but he forgot to do it.